config/configUtils.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs');
+
+const isFile = filePath => (fs.existsSync(filePath) && !fs.lstatSync(filePath).isDirectory());
+
+const dynamicRequire = (filePath) => {
+  if (isFile(filePath)) {
+    // eslint-disable-next-line global-require, import/no-dynamic-require
+    return require(filePath);
+  }
+  return undefined;
```

Comment:
Should we log something to the console if this condition is hit, more so to make the consumer aware the file they tried to require is not registering as a file.

Reply:
I could see value in it, but we have created this to attempt to load configuration for people from the root level, if it doesn't exist, they will need to add it themselves by extending the terra config for via the command line but this dynamic require may still run to attempt to find this file which would caused error console message that aren't helpful. IMO keeping this silent is better. Other's thoughts?